New website and logo announcement. (#45)
openwebdocs/owd-website: pages/content/posts/branding.md

---
author: "OWD team"
title: "Announcing the launch of our new logo and website"

> date: "2022-10-03"

Author: ```suggestion
date: "2022-10-04"
```

tags: ["announcements"]
ShowToc: false
ShowBreadCrumbs: false
excerpt: "Introducing a website redesign and a new logo, featuring Carle the bookworm"
---

Open Web Docs is pleased to announce the launch of a new website, a new logo, and Carle, the new Open Web Docs mascot.

Steering committee member and [Black Girls Tech](https://www.blackgirlsintech.org/) founder, [Lola Odelola](https://lolaodelola.dev/), led the OWD rebranding efforts. She coordinated the development of the new OWD mascot and logo and built the new website. 

Carle, pronounced *kahr-lee*, the OWD mascot, was designed by [Vladimir Grigoriev](https://nl.linkedin.com/in/grigza), a design lead at [JetBrains](https://Jetbrains.com). Carle is a bookworm! She’s named after the beloved children’s book author, [Eric Carle](https://eric-carle.com/). Carle personifies the OWD brand and is super memorable.  

> "I definitely love the new logo. It makes me happy every time I see it, and it perfectly reflects the marketing tone we started defining a year ago of fun, inclusion, accuracy, kindness, and goofiness. Hats off to Lola and Vladimir, and a huge thank you to JetBrains as well," remarked [Estelle Weyl](https://github.com/estelle), OWD staff writer. 

Reviewer: ```suggestion
"I definitely love the new logo. It makes me happy every time I see it, and it perfectly reflects the marketing tone we started defining a year ago of fun, inclusion, accuracy, kindness, and goofiness. Hats off to Lola and Vladimir, and a huge thank you to JetBrains as well!" remarked [Estelle Weyl](https://github.com/estelle), OWD staff writer. 
```

While Carle is usually smiling, she has five other facial reactions, enabling using slightly modified logos with different moods for different purposes. Her main color is a fantastic purple orchid, #B448FF.

> <img width="400" alt="Carle the bookworm with six different facial expressions" src="/assets/img/sixworms.png">

Author: ```suggestion
<img width="400" alt="Carle the bookworm with six different facial expressions" src="/img/sixworms.png">
```

> <img width="400" alt="Carle the bookworm with six different facial expressions" src="https://user-images.githubusercontent.com/69888/192943190-9b938307-4084-49e5-86c6-0b270e09f6ad.png">

Reviewer: Should we upload this image to a better place like `assets/img/`? Can you add it to this PR?

> <img width="400" alt="Carle the bookworm with six different facial expressions" src="https://user-images.githubusercontent.com/69888/192943190-9b938307-4084-49e5-86c6-0b270e09f6ad.png">

Author: ```suggestion
<img width="400" alt="Carle the bookworm with six different facial expressions" src="/assets/img/sixworms.png">
```

After a year of discussing a rebrand, we are so thankful to Lola and Vladimir for their hard work and dedication. We are delighted to officially announce the launch. The new site is available at the same URL as before: [OpenWebDocs.org](https://OpenWebDocs.org). Like everything OWD, the site is [open source and available on Github](https://github.com/openwebdocs/owd-website/). 

The new website provides our visitors with clear, easy way to [learn about Open Web Docs](https://openwebdocs.org/), who we are, what we do, and who funds us. You can learn about what we’ve accomplished and how you can [get involved](https://openwebdocs.org/get-involved/), including [sponsoring us and becoming a member](https://openwebdocs.org/membership/). We have a page about our [open source maintainers and steering and governing committees](https://openwebdocs.org/team/) and a page highlighting our [sponsors and funders](https://openwebdocs.org/sponsors/). The new website also includes [OWD's accomplishments for the past year](https://openwebdocs.org/content/reports/2021/) and a [blog with more details about many of our projects](https://openwebdocs.org/blog/). 

> We will continue to update our content with helpful information, articles, announcements, and project successes in the [blog](https://openwebdocs.org/content/posts/) section.

Reviewer: ```suggestion
We will continue to update our site with helpful information, articles, announcements, and project successes in the [blog](https://openwebdocs.org/content/posts/) section.
```

> We would like to Lola and Vladimir who gave their time, energy, and talents to make this site and our branding what it is.

Reviewer: ```suggestion
We would like to thank Lola and Vladimir who gave their time, energy, and talents to make this site and our branding what it is.
```

> For any questions, suggestions, feedback, or comments, please contact us on [Twitter](https://twitter.com/OpenWebDocs) or via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact).

Author: ```suggestion
For any questions, suggestions, feedback, or comments, please send us questions via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact), love on [Twitter](https://twitter.com/OpenWebDocs), website feedback via [github](https://github.com/openwebdocs/owd-website/ ) or content ideas via our [project proposal form](https://github.com/openwebdocs/project/issues/new?assignees=&labels=not+ready&template=owd-project.yml).
```

> For any questions, suggestions, feedback, or comments, please contact us on [Twitter](https://twitter.com/OpenWebDocs) or via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact).

Reviewer: Or maybe to file an issue on https://github.com/openwebdocs/owd-website/ for feedback?

Author: updated with four contact methods. Please review.

> For any questions, suggestions, feedback, or comments, please send us questions via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact), love on [Twitter](https://twitter.com/OpenWebDocs), website feedback via [github](https://github.com/openwebdocs/owd-website/) or content ideas via our [project proposal form](https://github.com/openwebdocs/project/issues/new?assignees=&labels=not+ready&template=owd-project.yml).

Reviewer: ```suggestion
For any questions, suggestions, feedback, or comments, please send us questions via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact), love on [Twitter](https://twitter.com/OpenWebDocs), website feedback via [GitHub](https://github.com/openwebdocs/owd-website/) or content ideas via our [project proposal form](https://github.com/openwebdocs/project/issues/new?assignees=&labels=not+ready&template=owd-project.yml).
```

> For any questions, suggestions, feedback, or comments, please send us questions via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact), love on [Twitter](https://twitter.com/OpenWebDocs), website feedback via [github](https://github.com/openwebdocs/owd-website/ ) or content ideas via our [project proposal form](https://github.com/openwebdocs/project/issues/new?assignees=&labels=not+ready&template=owd-project.yml).

Author: ```suggestion
For any questions, suggestions, feedback, or comments, please send us questions via the [Open Collective contact form](https://opencollective.com/open-web-docs/contact), love on [Twitter](https://twitter.com/OpenWebDocs), website feedback via [github](https://github.com/openwebdocs/owd-website/) or content ideas via our [project proposal form](https://github.com/openwebdocs/project/issues/new?assignees=&labels=not+ready&template=owd-project.yml).
```
